modules/core/src/test/java/org/apache/ignite/internal/processors/performancestatistics/AbstractPerformanceStatisticsTest.java

```diff
@@ -82,7 +82,7 @@ protected static void startCollectStatistics() throws Exception {
     }
 
     /** Stops and reads collecting performance statistics. */
-    protected static void stopCollectStatisticsAndRead(TestHandler... handlers) throws Exception {
```

Comment:
and here?

Reply:
Because I use this method in `RemoveConstantsFromQueryTest` to check that performance statistics will store queries without sensitive data